src/twisted/internet/_sslverify.py

```diff
@@ -1779,7 +1777,7 @@ def _expandCipherString(cipherString, method, options):
 
     @return: The effective list of explicit ciphers that results from the
         arguments on the current platform.
-    @rtype: L{list} of L{ICipher}
+    @rtype: L{tuple} of L{ICipher}
```

Comment:
May be a silly question, but it's worth asking: is changing lists for tuples necessary?

Reply:
Yes, because tuples are hashable and lists are not, and we need hashable types to make a low-overhead cache.

I'll update the docstrings.